src/main/java/de/tum/in/www1/artemis/web/rest/dto/ProgrammingExerciseTestCaseWeightUpdate.java

```diff
@@ -0,0 +1,14 @@
+package de.tum.in.www1.artemis.web.rest.dto;
+
+public class ProgrammingExerciseTestCaseWeightUpdate {
+
+    private Integer weight;
+
+    public Integer getWeight() {
+        return weight;
+    }
+
+    public void setWeight(Integer weight) {
+        this.weight = weight;
+    }
+}
```

Comment:
All other dto classes have DTO as a suffix.
The class name is too long in my opinion and doesn't represent it's functionality, which is only to store a weight.
I would suggesst `WeightDTO` or `TestCaseWeightDTO`

Reply:
I prefixed ProgrammingExercise to make sure everybody understands it's domain specific.
The alternative would be to move it into a folder within the DTO directory :thinking:

Reply:
But  why does this class need to be specific for ProgrammingExerciseWeights?
In my opinion, it would be totally fine to store other types of weights in it.

Reply:
I changed it :+1: